src/extension/gogoduck/pom.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>au.org.emii.wps</groupId>
+    <artifactId>gogoduck</artifactId>
+    <version>2.0</version>
+
+    <parent>
+        <groupId>org.geoserver</groupId>
+        <artifactId>extension</artifactId>
+        <version>2.7.0</version>
+    </parent>
+
+    <dependencies>
```

Comment:
I think we're going to end up with a lot of duplication in the POM between the root POM and all of the `extension/*/pom.xml` files.  We should have a look at refactoring that once everything has been brought in to this repo.